container/ubuntu16_04/layers/bazel/BUILD

```diff
@@ -26,22 +29,38 @@ language_tool_layer(
     name = "bazel-tools",
     base = "@ubuntu16_04//image",
     packages = [
+        "bash-completion",
+        "binutils",
         "patch",
         "unzip",
         "zip",
     ],
 )
 
-language_tool_layer(
-    name = "bazel-ltl",
-    additional_repos = [
-        "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8",
-    ],
+[toolchain_container(
```

Comment:
add short doc to each of these blocks indicating this produces a target of certain type per bazel version

Reply:
Done.